packages/mobile/src/import/saga.test.ts

```diff
@@ -51,31 +77,81 @@ describe('Import wallet saga', () => {
     await expectSuccessfulSagaWithPhrase(mockValidSpanishPhrase)
   })
 
-  const expectFailedSagaWithPhrase = async (phrase: string) => {
+  it('fails for a phrase invalid checksum', async () => {
```

Comment:
I thought we didn't implement the checksum?
From what I see the phrase is still technically valid, but there's no account with a balance with it.
I think we should still allow it and ask the user if they still really want to use it.
What do you think?

Reply:
BIP-39 implements a short checksum and we use it. This phrase does indeed have an invalid checksum, otherwise they would get that option.